src/eko/matching_conditions/operator_matrix_element.py

```diff
@@ -207,7 +207,7 @@ def quad_ker(u, order, mode, is_log, logx, areas, a_s, L, backward_method):
 
     # select the need matrix element
     ker = ker[indeces[mode[-2]], indeces[mode[-1]]]
-    if ker == 0.0:
+    if np.real(ker) == 0.0:
```

Comment:
that seems wrong ... why do you want to skip at all? `ker` can be purely imaginary and still contribute to the integral ...

Reply:
Indeed I forgot to remove this